### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by GlendaChong**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -316,75 +316,72 @@ If you wish to learn more about the other features of LinkMeIn, head over to our
...
+* For commands that do not take in any parameters (such as help, list, exit and clear), anything typed after the command word will be ignored. For example, if you type `help 123`, `123` will be ignored, and the command will just be interpreted as `help`.
+* If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines! It could result in an invalid input.
 </div>
 
 ## General Features
-In this section, you'll find essential tools that empower you to navigate LinkMeIn with ease. Learn how to access help, 
-clear all entries, and gracefully exit the application, ensuring a seamless user experience.
 
-### Clear All Companies: `clear`
+### Clearing All Companies: `clear`
```
Comment (tayruxin) [inline docs/UserGuide.md:340]: was the hyperlink for this edited in quick tour. There is a hyperlink in quick tour that links to this header, might need to update the hyperlink

File: docs/UserGuide.md [hunk 2]
```diff
@@ -469,136 +461,121 @@ You can use the delete command to delete the company from the list of companies
...
 
 **Format:** `remark INDEX re/REMARK`
 
-> `INDEX`: Adds remarks to the company at the INDEX number shown in the displayed company list.  
-> `re/REMARK` : Type prefix `re/` followed by the remarks you want to add. 
-
-<div markdown="block" class="alert alert-info">
-**:information_source: Note:**<br>
-* Remarks cannot be empty. If you wish to remove existing remarks, use the  [`unremark` command](#deleting-remarks-of-a-company) instead.
-</div>
```
Comment (tayruxin) [inline docs/UserGuide.md:501]: I think this makes sense tho? Maybe can be a tip instead? Under the possible error section

Comment (GlendaChong) [reply docs/UserGuide.md:501]: okay, will add that in

**Review by tayruxin — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -402,203 +399,188 @@ Add the company to LinkMeIn now to keep track of the application process using t
...
+
+<div markdown="block" class="alert alert-success">
+**:bulb: Useful Tip:**<br>
+* However, if you intend to delete the existing remarks of a company and hence entered an empty remark, you should use the `unremark` command instead.
+</div>
 
 ### Deleting Remarks of a Company: `unremark`
 
-_You have previously added some remarks to a company, but the remark has become outdated and you no longer need them anymore. Use the `unremark` command to quicly delete it._
+_You have previously added some remarks to a company, but the remarks has become outdated and you no longer need them anymore. Use the `unremark` command to quickly delete them._
```
Comment (papataco14) [inline docs/UserGuide.md:524]: but the remarks "have" become

**Review by papataco14 — CHANGES_REQUESTED** [review]
grammar and suggestions

File: docs/UserGuide.md (hunk 3 above)
Comment (GlendaChong) [reply docs/UserGuide.md:524]: edited

**Review by papataco14 — APPROVED** [review]
LGTMMMMMMMM

**Pull request merged**